data/categories_cuisines.txt

```diff
@@ -839,6 +863,7 @@ nl:Steakhouse
 fi:Pihviravintola
 fr:Grillades
 de:Steakhaus
+hi:स्टेक हाउस
```

Comment:
```suggestion
hi:स्टेक (माँस का कबाब)
```

Reply:
@rtsisyk is it ok to use parenthesis here?